pkg/apis/discovery/validation/validation.go

```diff
@@ -40,7 +40,7 @@ var (
 	)
 	maxTopologyLabels = 16
 	maxAddresses      = 100
-	maxPorts          = 100
+	maxPorts          = 16000
```

Comment:
Despite the small diff, there is test coverage for this, it just uses this variable.

Reply:
https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/discovery/validation/validation_test.go#L269 and https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/discovery/validation/validation_test.go#L168